scrapping/scrapping/spiders/books.py

```diff
@@ -0,0 +1,72 @@
+import scrapy
+from scrapy import Selector
+from scrapy.http import Response
+from selenium import webdriver
+
+
+class BookSpider(scrapy.Spider):
+    name = "book"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
+
+    def __init__(self, **kwargs) -> None:
+        super().__init__(**kwargs)
+        self.driver = webdriver.Chrome()
+
+    def close(self, reason: str) -> None:
+        self.driver.close()
+
+    def parse(self, response: Response, **kwargs) -> dict:
+        for book in response.css(".product_pod"):
+            info = self._parse_additional_info(response, book)
+            yield {
+                "title": book.css("a::attr(title)").get(),
+                "price": book.css(".price_color::text").get(),
+                "amount_in_stock": info["amount_in_stock"],
+                "rating": info["rating"],
+                "category": info["category"],
+                "description": info["description"],
+                "upc": info["upc"],
+            }
```

Comment:
Ensure that the scraped data is saved to a `books.jl` file as required by the task. You can use Scrapy's feed export feature to achieve this.